pom.xml

```diff
@@ -1,43 +1,54 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-    <groupId>de.uni-leipzig.life.csv2fhir</groupId>
-    <artifactId>csv2fhir</artifactId>
-    <packaging>jar</packaging>
-    <version>1.0-SNAPSHOT</version>
+	<groupId>de.uni-leipzig.life.csv2fhir</groupId>
```

Comment:
Wir verwenden üblicherweise keine Tabs. Es wäre schön, wenn Du das bei Dir in Eclipse anpassen könntest.